templates/partials/committee-charts.html

```diff
@@ -1,3 +1,6 @@
+{% with %}
```

Comment:
This partial is included in both candidate and committee pages-- it works for one, but not the other. The two need to be reconciled. I don't think just removing the `with` will do it. (I don't have the answer yet, though.)